components/builder-web/bs-config.js

```diff
@@ -0,0 +1,13 @@
+const proxyMiddleware = require('http-proxy-middleware');
```

Comment:
Are there any specific steps for devs to take advantage of this?

Reply:
This makes me wonder whether the API URL even needs to be configurable anymore. I don't think it does, so I'll update this PR to remove that as well.